src/python/foglamp_daemon.py

```diff
@@ -17,7 +20,6 @@ def do_something(logf):
     fh.setFormatter(formatter)
```

Comment:
I know this is unrelated but "fh" variable needs to be changed and I vote for meaningful variable name. Also, I see this warning in pylint_* log file. It would be great if we fix warnings as per pylinter check.